curriculum/challenges/_meta/lab-date-conversion/meta.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Build a Date Conversion Program",
+  "isUpcomingChange": true,
+  "usesMultifileEditor": true,
+  "dashedName": "lab-date-conversion",
+  "order": 219,
+  "superBlock": "front-end-development",
+  "challengeOrder": [{ "id": "66f686b8ebdb982fa8e14330", "title": "Step 1" }],
```

```suggestion
  "challengeOrder": [{ "id": "66f686b8ebdb982fa8e14330", "title": "Build a Date Conversion Program" }],
```

I don't think this gets used or displayed, but might as well fix it.